### DISCUSSION TITLE
ng-mocks & Directive composition API

### DISCUSSION THREAD

**Discussion by LukasMachetanz** [help-me-mom/ng-mocks#5093] [Q&A] [answered]
Hey!

Is it possible to mock directives, which are applied using the [Directive composition API](https://angular.io/guide/directive-composition-api)?

At the moment of writing it seems that I can't get it to work. Any help appreciated.

Greets, 
Lukas

**Comment by LukasMachetanz** [comment, marked as answer]
@satanTime, is this somewhere mentioned within the documentation?

**Reply by satanTime** [reply]
I've create an FR, thank you for the idea: https://github.com/help-me-mom/ng-mocks/issues/5134
